python/nano/tutorial/notebook/inference/pytorch/inference_optimizer_optimize.ipynb

```diff
@@ -222,17 +229,302 @@
+   "metadata": {},
+   "source": [
+    "In some cases, if you expect that some acceleration methods will not work for your model / not work well / run for too long / cause exceptions to the program, you could avoid running these methods by specifying `excludes` paramater:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "optimizer.optimize(model=model,\n",
+    "                   training_data=train_dataloader,\n",
+    "                   validation_data=val_dataloader,\n",
+    "                   metric=accuracy,\n",
+    "                   direction=\"max\",\n",
+    "                   thread_num=1,\n",
+    "                   excludes=[\"onnxruntime_int8_qlinear\", \"openvino_int8\"],\n",
+    "                   latency_sample_num=100)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The example output of `optimizer.optimize` is shown below.\n",
+    "\n",
+    "```bash\n",
+    " -------------------------------- ---------------------- -------------- ----------------------\n",
+    "|             method             |        status        | latency(ms)  |       accuracy       |\n",
+    " -------------------------------- ---------------------- -------------- ----------------------\n",
+    "|            original            |      successful      |    31.872    |        0.794         |\n",
+    "|              bf16              |      successful      |    17.326    |        0.794         |\n",
+    "|          static_int8           |      successful      |    12.39     |        0.786         |\n",
+    "|         jit_fp32_ipex          |      successful      |    18.871    |        0.794*        |\n",
+    "|  jit_fp32_ipex_channels_last   |      successful      |    18.453    |        0.794*        |\n",
+    "|         jit_bf16_ipex          |      successful      |    9.863     |        0.792         |\n",
+    "|  jit_bf16_ipex_channels_last   |      successful      |    9.871     |        0.792         |\n",
+    "|         openvino_fp32          |      successful      |    24.585    |        0.794*        |\n",
+    "|        onnxruntime_fp32        |      successful      |    19.452    |        0.794*        |\n",
+    " -------------------------------- ---------------------- -------------- ----------------------\n",
+    "* means we assume the precision of the traced model does not change, so we don't recompute accuracy to save time.\n",
+    "Optimization cost 53.2s in total.\n",
+    "```"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4. Disable validation during optimize\n",
+    "\n",
+    "If you can't get corresponding validation dataloader for you model, or you don't care about the possible accuracy drop, you could just disable validation by don't specify `validation_data`, `metric`, `direction` paramater:"
```

Comment:
change 'you could just disable validation by don't specify `validation_data`, `metric`, `direction` paramater:' to 'you can omit `validataion_data`, `metrice` and `direction` parameters to disable validation'

Reply:
done